work_queue/src/work_queue_graph_log

```diff
@@ -47,6 +47,19 @@ def read_log_entries(file, fields):
     pairs = zip(idxs, fields)
     epoch = None
     count_lines = 0
+    prev_line = {}
```

Comment:
To make it easier, I would do here  prev_line = None
and below at line 90:

if prev_line is None:
    prev_line = record